libs/shared/flagd-core/package.json

```
@@ -1,13 +1,16 @@
 {
   "name": "@openfeature/flagd-core",
-  "version": "0.0.1-alpha",
+  "version": "0.1.0",
   "scripts": {
     "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
     "current-version": "echo $npm_package_version"
   },
   "dependencies": {
-    "@openfeature/server-sdk": ">=1.6.0",
+    "@openfeature/core": "^0.0.16",
     "ajv": "^8.12.0",
     "tslib": "^2.3.0"
+  },
+  "devDependencies": {
+    "@types/imurmurhash": "^0.1.3"
```

```suggestion
  },
  "devDependencies": {
    "@types/imurmurhash": "^0.1.3"
```

This will be computed automatically by nx

---

removed